lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -108,12 +123,20 @@ export class RefreshTokenClient extends BaseClient {
      */
     private async acquireTokenWithCachedRefreshToken(request: CommonSilentFlowRequest, foci: boolean) {
         // fetches family RT or application RT based on FOCI value
+
+        // @ts-ignore
+        const atsMeasurement = this.performanceClient?.startMeasurement("RefreshTokenClientAcquireTokenWithCachedRefreshToken", request.correlationId);
```

Comment:
This can also be added to the enum (lets always add to the enum except for in the 1p repo).

Reply:
Got it, thanks . Have added both to the enum.